parity/cli/mod.rs

```diff
@@ -482,6 +482,10 @@ usage! {
 			"--jsonrpc-server-threads=[NUM]",
 			"Enables multiple threads handling incoming connections for HTTP JSON-RPC server.",
 
+			ARG arg_jsonrpc_max_payload: (Option<usize>) = None, or |c: &Config| c.rpc.as_ref()?.max_payload,
+			"--jsonrpc-max-size=[NUM]",
```

Comment:
should be `MB` instead of `NUM`

Reply:
Thanks for looking over my PR. I've added both of your suggestions.